tools/workshop/generate-profiles-from-seeds.ps1

```
@@ -12,129 +11,128 @@ $ErrorActionPreference = "Stop"
-        parentProfile = $baseProfile
-        profileAliases = ($aliases | Select-Object -Unique) -join ","
-        localPathHints = ($modPathHints | Select-Object -Unique) -join ","
-        requiredWorkshopIds = ($steamModIds | Select-Object -Unique) -join ","
+    foreach ($cap in @(Get-StringList -Value $seed.requiredCapabilities)) {
+        if (-not $requiredCapabilities.Contains($cap)) {
+            $requiredCapabilities.Add($cap)
+        }
     }
 
-    $profilePayload = [ordered]@{
+    $profile = [ordered]@{
```

PSScriptAnalyzer (1.24.0)

[warning] The Variable 'profile' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)